services/worker-manager/src/providers/google.js

```diff
@@ -15,7 +15,7 @@ class GoogleProvider extends Provider {
     ...conf
   }) {
     super(conf);
-    let {project, creds, workerServiceAccountId, apiRateLimits = {}} = providerConfig;
+    let {project, creds, workerServiceAccountId, apiRateLimits = {}, _backoffDelay = 1000} = providerConfig;
```

Comment:
Would this have someone deploying TC putting `_backoffDelay` into their `PROVIDERS`?

Reply:
Ideally they won't set it. I just wanted it for testing since the faketime stuff doesn't work here. I prefixed it with the `_` and left it undocumented.

Reply:
Ah, got it.